controller/organization.js

```diff
@@ -0,0 +1,60 @@
+import {
+    addNewOrganization, deleteOrganizationById, updateOrganizationById, getOrganization,
```

Comment:
The function to get organization as defined in `services/organization.js` is named `getOrganizations` and not `getOrganization`